.circleci/config.yml

```diff
@@ -696,8 +679,8 @@ jobs:
               git config --local user.name "hc-github-team-consul-core"
 
               short_sha=$(git rev-parse --short HEAD)
-              git add agent/uiserver/bindata_assetfs.go
-              git commit -m "auto-updated agent/uiserver/bindata_assetfs.go from commit ${short_sha}"
+              git add agent/uiserver/dist/
```

Comment:
Now that we have multiple files and directories, not just a single file, I think we'll need to first `git rm -r agent/uiserver/dist` before we build anything to remove any stale files, right?

Otherwise we'll never be able to prune files. And I suspect since many of the files have fingerprints in the name we'd end up with a new copy of all the files every time.

Or is that remove done somewhere else that I missed?

In the removed job we have `mv pkg/packages/consul-ui/dist pkg/web_ui`. Do we need something similar to move these files over?

Reply:
Ah good catch! I thought my changes in `build-support/functions/20-build.sh` would move the files but we don't run any `make` commands that end up calling that function

Reply:
Now I'm a little confused as to how those build-support/ scripts relate to our CircleCI workflows... 😕